src/js/chromeAppEventPage.js

```diff
@@ -0,0 +1,27 @@
+/*
+    If an id is also specified and a window with a matching id has been shown before, the remembered bounds of the window will be used instead.
+*/
+'use strict';
+
+function startApplication() {
+    chrome.app.window.create('main.html', {
+        id: 'main-window',
+        frame: 'chrome',
+        innerBounds: {
+            minWidth: 1024,
+            minHeight: 550,
+        },
+    }, function (createdWindow) {
+        if (getChromeVersion() >= 54) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['getChromeVersion' was used before it was defined.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5573000)